projects/demo/src/modules/components/input-range/input-range.component.ts

```diff
@@ -18,9 +19,10 @@ import {ABSTRACT_PROPS_ACCESSOR} from '../abstract/inherited-documentation/abstr
             provide: ABSTRACT_PROPS_ACCESSOR,
             useExisting: forwardRef(() => ExampleTuiInputRangeComponent),
         },
+        tuiDocExcludeProperties(['decimal', 'decimalLimit']),
```

Comment:
`zeroPadding` do nothing for `InputRange`.
Should we exclude it too ?

https://taiga-ui--pr6997-number-format-direct-u23z2z12.web.app/components/input-range/API?zeroPadding=false&quantum=0.001

Reply:
The same question for `InputSlider` page